ios/ru.lproj/Localizable.strings

```diff
@@ -1,29 +1,29 @@
-"no-favorites" = "No Favorites";
-"no-favorites-message" = "You can add favorites from your iPhone";
-"no-trains-found" = "No trains found for today";
+"no-favorites" = "Нет избранных маршрутов";
+"no-favorites-message" = "Добавить маршрут в избранное можно с iPhone";
+"no-trains-found" = "Поездов на сегодня не найдено";
 
-"departure" = "Depart";
-"arrival" = "Arrive";
+"departure" = "Отправление";
+"arrival" = "Прибытие";
 
-"train %@" = "Train %@";
-"platform %@" = "Platform %@";
-"train %@ platform %@" = "Train %@ ・ Platform %@";
+"train %@" = "Поезд %@";
+"platform %@" = "Платформа %@";
+"train %@ platform %@" = "Поезд %@ ・ Платформа %@";
 
-"exchange" = "Exchange";
-"stay-at %@" = "Stay at Platform %@";
-"move-to %@" = "Move to Platform %@";
+"exchange" = "Пересадка";
+"stay-at %@" = "Остаться на платформе %@";
+"move-to %@" = "Перейти на платформу %@";
 
-"NEXT TRAIN" = "הרכבת הבאה";
-"UPCOMING" = "הרכבות הבאות";
-"TOMORROW" = "מחר";
-"ARRIVAL" = "הגעה";
-"PLATFORM" = "רציף";
-"TRAIN NO." = "רכבת מס׳";
-"TRAIN" = "רכבת";
+"NEXT TRAIN" = "Следующий поезд";
+"UPCOMING" = "Следующие поезда";
+"TOMORROW" = "Завтра";
+"ARRIVAL" = "Прибытие";
+"PLATFORM" = "Платформа";
+"TRAIN NO." = "Поезд номер";
```

Comment:
Is there a shorter string for this?
<img width="369" alt="Screen Shot 2022-03-10 at 10 29 00" src="https://user-images.githubusercontent.com/13344923/157620381-26b8f8be-e8fd-4127-8f57-35aa2c301218.png">